include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -40,13 +40,12 @@ struct builtin_atomic {
 namespace detail
 {
 
-#if defined(RAJA_COMPILER_MSVC)
+#if defined(RAJA_COMPILER_MSVC) || defined(_WIN32)
```

Comment:
I missed this the first time through, but shouldn't this be a check for ICC on windows? Clang on windows does not have this problem, mingw probably doesn't either.

Reply:
This has been fixed.